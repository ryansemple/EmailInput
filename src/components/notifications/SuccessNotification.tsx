import React from "react";
import Notification from "./Notification";

interface SuccessNotificationProps {
	text: string
}

/**
 * Renders a success notification.
 */
const SuccessNotification = (props: SuccessNotificationProps) => (
	<Notification 
		text={props.text}
		className="Notifications_Notification-Success"
	/>
)

export default SuccessNotification;