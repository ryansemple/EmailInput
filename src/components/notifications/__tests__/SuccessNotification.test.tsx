import React from "react";
import { render } from "@testing-library/react";
import SuccessNotification from "../SuccessNotification";

test("Renders SuccessNotification component correctly.", () => {
	const { container } = render(
		<SuccessNotification 
			text="Success"
		/>
	);
	expect(container).toMatchSnapshot();
});