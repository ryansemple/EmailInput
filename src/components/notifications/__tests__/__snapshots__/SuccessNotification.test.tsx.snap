// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders SuccessNotification component correctly. 1`] = `
<div>
  <div
    class="Notifications_Notification Notifications_Notification-Success"
    role="alert"
  >
    <p>
      Success
    </p>
  </div>
</div>
`;
